### ISSUE TITLE
Change the behaviour of the config file

### ISSUE THREAD

**Issue by scripthunter7** [AdguardTeam/AGLint#116]
- CLI should give an error if there is no config file in the root folder
- VSCode extension linter feature shouldn't run if there is no config file in the root folder
- We need to create an "aglint init" command that will create a config file that will extend the default config as follows:
  ```yml
  extends:
    - aglint:recommended
  ```

**Comment by scripthunter7** [comment]
Completed in [v1.0.11](https://github.com/AdguardTeam/AGLint/releases/tag/v1.0.11)
